extensions/logging-json/runtime/src/main/java/io/quarkus/logging/json/runtime/JsonConfig.java

```diff
@@ -51,4 +54,22 @@
      */
     @ConfigItem
     boolean printDetails;
+    /**
+     * Override keys with custom values. The special string "default" indicates that no key overrides will be applied.
+     */
+    @ConfigItem(defaultValue = "default")
+    String keyOverrides;
```

Comment:
Maybe it would make more sense for this to be an `Optional<String>` rather than using a special value.

Reply:
Agree, that's better, done 👍